azurerm/resource_arm_api_management.go

```diff
@@ -627,7 +628,7 @@ func resourceArmApiManagementServiceRead(d *schema.ResourceData, meta interface{
 	}
 
 	policyClient := meta.(*ArmClient).ApiManagement.PolicyClient
-	policy, err := policyClient.Get(ctx, resourceGroup, name)
+	policy, err := policyClient.Get(ctx, resourceGroup, name, apimanagement.PolicyExportFormatXML) // TODO: Verify this setting
```

Comment:
Need help deciding on which format to use. There are two choices:
```go
// PolicyExportFormatRawxml The contents are inline and Content type is a non XML encoded policy document.
PolicyExportFormatRawxml PolicyExportFormat = "rawxml"
// PolicyExportFormatXML The contents are inline and Content type is an XML document.
PolicyExportFormatXML PolicyExportFormat = "xml"
```

Reply:
`xml` would be the right thing to do, because otherwise it will always plan a ~change.
If you use `rawxml` When comparing the state, it will always find a difference between the readable indented xml and the raw inline export

Reply:
Thanks for helping out on this one :) I'll mark the other related issues as resolved.